test/unit/event/send-acknowledgement-event.test.js

```diff
@@ -0,0 +1,56 @@
+const mockFRN = require('../../mock-frn')
+const sendAcknowledgementEvent = require('../../../app/event/send-acknowledgement-event')
+
+jest.mock('../../../app/event/send-processing-ack-error-event')
+const sendProcessingAckErrorEvent = require('../../../app/event/send-processing-ack-error-event')
+
+jest.mock('../../../app/event/send-invalid-bank-details-event')
+const sendInvalidBankDetailsEvent = require('../../../app/event/send-invalid-bank-details-event')
+
+let mockAcknowledgementError
+let mockHoldCategoryName
+
+describe('send acknowledgement event', () => {
+  beforeEach(() => {
+    mockAcknowledgementError = require('../../mock-acknowledgement-error')
+    mockHoldCategoryName = 'DAX rejection'
```

Comment:
Could you add a new folder called `constants` and file `hold-categories-names.js` with both `'DAX rejection'` and `'Bank account anomaly'`. We're using them more than once throughout this application, let's standardise them!

Reply:
Similar to constant folder we have in other repos, e.g. https://github.com/DEFRA/ffc-pay-statement-constructor/tree/main/app/constants